design-toolkit/components/src/index.css

```diff
@@ -15,218 +15,9 @@
-  --alpha-white-02: rgba(255, 255, 255, 0.4);
-  --alpha-white-03: rgba(255, 255, 255, 0.8);
-
-  --highlight-primary-01: #c9fdef;
-  --highlight-primary-02: #7df9d8;
-  --highlight-primary-03: #28f5be;
-  --highlight-primary-04: #189372;
-  --highlight-primary-05: #0a3d30;
-
-  --highlight-secondary-01: #c9f6fd;
-  --highlight-secondary-02: #7ee9f9;
-  --highlight-secondary-03: #28dcf5;
-  --highlight-secondary-04: #188493;
-  --highlight-secondary-05: #0a373d;
-
-  --highlight-tertiary-01: #f9c5e2;
-  --highlight-tertiary-02: #f174b9;
-  --highlight-tertiary-03: #e8178a;
-  --highlight-tertiary-04: #8a0e53;
-  --highlight-tertiary-05: #3a0623;
-
-  --utility-info-01: #e6e6e6;
-  --utility-info-02: #898989;
-  --utility-info-03: #686868;
-  --utility-info-04: #424244;
-  --utility-info-05: #222222;
-
-  --utility-advisory-01: #e0edff;
-  --utility-advisory-02: #a1caff;
-  --utility-advisory-03: #62a6ff;
-  --utility-advisory-04: #3a6499;
-  --utility-advisory-05: #142133;
-
-  --utility-normal-01: #d6f6e5;
-  --utility-normal-02: #82e4b2;
-  --utility-normal-03: #30d27e;
-  --utility-normal-04: #1c7e4c;
-  --utility-normal-05: #0a2a19;
-
-  --utility-serious-01: #feedcc;
-  --utility-serious-02: #fdc865;
-  --utility-serious-03: #fca400;
-  --utility-serious-04: #976201;
-  --utility-serious-05: #322100;
-
-  --utility-critical-01: #f6d3d2;
-  --utility-critical-02: #e57b77;
-  --utility-critical-03: #d4231d;
-  --utility-critical-04: #7f1511;
-  --utility-critical-05: #2a0706;
-
-  --classification-unclass: #007A33;
-  --classification-cui: #502b85;
-  --classification-confidential: #0033a0;
-  --classification-secret: #c8102e;
-  --classification-top-secret: #ff8c00;
-  --classification-ts-sci: #fce83a;
-
-  --header-xxl-size: 32px;
-  --header-xxl-height: 40px;
-  --header-xl-size: 24px;
-  --header-xl-height: 28px;
-  --header-l-size: 20px;
-  --header-l-height: 24px;
-  --header-m-size: 14px;
-  --header-m-height: 20px;
-  --header-s-size: 12px;
-  --header-s-height: 16px;
-  --header-xs-size: 10px;
-  --header-xs-height: 12px;
-
-  --body-l-size: 16px;
-  --body-l-height: 24px;
-  --body-m-size: 14px;
-  --body-m-height: 20px;
-  --body-s-size: 12px;
-  --body-s-height: 16px;
-  --body-xs-size: 10px;
-  --body-xs-height: 12px;
-  --body-xxs-size: 9px;
-  --body-xxs-height: 12px;
-
-  --button-l-size: 16px;
-  --button-l-height: 24px;
-  --button-m-size: 14px;
-  --button-m-height: 20px;
-  --button-s-size: 12px;
-  --button-s-height: 16px;
-  --button-xs-size: 10px;
-  --button-xs-height: 12px;
-}
+/* Import generated themes.css */
+@import './tokens/generated/themes.css';
 
-@theme {
-  /* Remove the default Tailwind styles. */
-  --color-*: initial;
-  --font-*: initial;
-  --radius-*: initial;
-  --shadow-*: initial;
-  --spacing-*: initial;
-  --text-*: initial;
-
-  /** Fonts **/
-  --font-primary: "Roboto Flex", sans-serif;
-  --font-display: "Roboto Mono", monospace;
-
-  /** Colors **/
-  --color-current: currentColor;
```

Comment:
This color var is missing from the generated output. Can you please go through and check again that there aren't anymore missing?

Reply:
sweet added and double checked all others;